project-koku/koku
# fix updating cluster_alias

## Jira Ticket

[COST-3793](https://issues.redhat.com/browse/COST-3793)

## Description

This change will check if an entry exists for a provider in the reporting_ocp_clusters table before creating one, if the cluster_alias is different we will also update that.

## Testing

1. Checkout Branch
2. Restart Koku
3. create/load test data
4. update a source/provider name
5. see the cluster alias is update correctly in reporting_ocp_clusters table.

## Notes

...
